### PR TITLE
Allow bypass elasticsearch certificate validation

### PR DESCRIPTION
**Pull request by horacimacias**

### CODE REVIEW

**Comment by tsaikd** [comment]
Can you add an unit test?

**Comment by horacimacias** [comment]
I've renamed the vars based on your suggestions. I'm not sure why the travis job is failing; can this be related to migration to modules?

File: output/elastic/README.md [hunk 1]
```diff
@@ -50,4 +50,9 @@ output:
     # (optional) default: "5m"
     # ExponentialBackoffMaxTimeout used to set the maximum wait interval in elastic.ExponentialBackoff
     exponential_backoff_max_timeout: "5m"
+
+    # (optional) default: "true"
+    # SslCertValidation Option to validate the server's certificate. Disabling this severely compromises security.
```
Comment (tengattack) [inline output/elastic/README.md]: Don't forget the document :)

**Comment by tengattack** [comment]
I think tsaikd will solve travis job's problem before merging or later :)

File: output/elastic/README.md (hunk 1 above)
Comment (horacimacias) [reply output/elastic/README.md]: ah sorry. done

File: output/elastic/outputelastic_test.go [hunk 2]
```diff
@@ -20,6 +23,70 @@ func init() {
 	config.RegistOutputHandler(ModuleName, InitHandler)
 }
 
+func Test_SSLCertValidation(t *testing.T) {
+	a := assert.New(t)
```
Comment (tsaikd) [inline output/elastic/outputelastic_test.go]: please use `assert := assert.New(t)` to keep code consistency

File: output/elastic/outputelastic_test.go [hunk 3]
```diff
@@ -20,6 +23,70 @@ func init() {
...
+	// check default config is 'true'
+	a.True(DefaultOutputConfig().SSLCertValidation, "Default ssl validation must be true")
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		fmt.Printf("%v\n", r)
```
Comment (tsaikd) [inline output/elastic/outputelastic_test.go]: necessary?

**Comment by tsaikd** [comment]
Please squash your commits

**Comment by horacimacias** [comment]
Let me try again on a different branch as I messed up.
